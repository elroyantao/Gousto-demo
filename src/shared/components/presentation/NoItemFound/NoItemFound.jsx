import React from 'react'
import styles from './NoItemFound.css'

export default function NoItemFound() {
  return (
    <div className={styles.NoItemFound}>
      <p> No items found </p>
    </div>
  )
}
