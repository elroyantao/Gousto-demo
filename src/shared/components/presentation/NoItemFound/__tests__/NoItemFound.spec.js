import testComponentHelper from '../../../../lib/test-helper'
import NoItemFound from '../NoItemFound'

describe('<NoItemFound/>', () => {
  const renderComponent = testComponentHelper(NoItemFound)

  describe('@renders', () => {
    it('in default state', () => {
      expect(renderComponent({}).getTree())
        .toMatchSnapshot()
    })
  })
})
