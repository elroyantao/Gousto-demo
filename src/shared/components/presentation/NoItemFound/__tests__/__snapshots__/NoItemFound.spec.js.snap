// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoItemFound/> @renders in default state 1`] = `
<div
  className="NoItemFound"
>
  <p>
     No items found 
  </p>
</div>
`;
